docs/source/installation/playground.rst

```diff
@@ -54,15 +51,12 @@ Major Services in Playground
    * - ``Redis Commander``
      - Redis web GUI.
      - http://127.0.0.1:40009
-   * - ``Read the Docs``
-     - Local host docs.
-     - http://127.0.0.1:40010
    * - ``Jupyter Lab``
      - Jupyter lab with MARO environment, examples, notebooks.
-     - http://127.0.0.1:40011
+     - http://127.0.0.1:40010
 
 
-*(If you use other port mapping, remember to change the port number.)*
+*(Remember to change ports, if you use different ports mapping.)*
```

Comment:
Remember to change ports, if you use different ports mapping.
->
Remember to change ports if a different port mapping is used.

Reply:
done